hack/manifests/crds.go

```diff
@@ -21,11 +21,28 @@ func cleanCRD(filename string) {
 		patchWorkflowSpecTemplateFields(&schema, "status", "properties", "storedWorkflowTemplateSpec", "properties")
 		patchTemplateFields(&schema, "status", "properties", "storedTemplates", "additionalProperties", "properties")
 		patchEphemeralVolumeFields(&schema, "status", "properties", "persistentVolumeClaims", "items", "properties")
+		// Even with server-side apply, trying to apply
+		// "manifests/base/crds/full/argoproj.io_workflows.yaml" results in
+		// either the error "Request entity too large: limit is 3145728" or the
+		// error "Error from server: rpc error: code = ResourceExhausted desc =
+		// trying to send message larger than max (2163613 vs. 2097152)".
```

Comment:
Just want to double check our current test suite would capture this error, right? (so that we don't need to add extra tests to check the crd size)
e.g. if we didn't have the optimisation, the test suite would fail due to being unable to install the crd?